package.json

```diff
@@ -4298,6 +4298,7 @@
         "c8": "^9.0.0",
         "circular-dependency-plugin": "^5.2.2",
         "css-loader": "^6.7.3",
+        "dependency-cruiser": "^16.0.0",
```

Comment:
Is this actually needed? The doc mentions `npx`, which should automatically pull stuff dynamically.

Reply:
Makes sense, I've removed the dependency